docs/appendices/release-notes/5.4.4.rst

```diff
@@ -60,3 +60,7 @@ Fixes
 - Fixed a regression introduced with CrateDB 5.3.1 which caused a
   ``NullPointerException`` when creating a repository on S3 with authentication
   provided by IAM role attached to the EC2 instance running the CrateDB node.
+
+- Fixed an issue which led to skipping duplicate column check, in
+  ``CREATE TABLE`` statements, if duplicate columns have the same type
+  with already defined one.
```

Comment:
apologies, with the rewording, this line is not needed at all.

Reply:
https://github.com/crate/crate/pull/14752/commits/a8c4b421b2fce9fe98c4b4f31a28bc22e9124414

Could you please have another look?